src/App.js

```diff
@@ -18,15 +18,20 @@ function App() {
   const [todoList, setTodoList] = useSemiPersistentState();
 
   const addTodo = (newTodo) => {
-    setTodoList([...todoList, newTodo]);
+    setTodoList((previousTodoList) => [...previousTodoList, newTodo]);
+  };
+
+  const removeTodo = (id) => {
+    const handleRemove = todoList.filter((todo) => id.title !== todo.title);
+    setTodoList(handleRemove);
```

Comment:
Same issue here as we had before in the `addTodo`. We need the previous state to create the new state. Try to figure this out on your own 😄 

I know you can do it 🎖️